boefjes/tests/modules/dummy_normalizer_import_error/normalize.py

```diff
@@ -0,0 +1,7 @@
+from octopoes.bad import wrong  # noqa
+
+
+def run(normalizer_meta: NormalizerMeta, raw: Union[bytes, str]) -> Iterator[OOI]:  # noqa
+    network = Network(name=raw.decode())  # noqa
+
+    yield network  # noqa
```

Comment:
Does this even matter if the interpreter already fails on the first line?

Reply:
You mean the `# noqa`?

Reply:
I mean the entire function block, because it's dead code. It will never be evaluated so that's confusing and adding `noqa` on every line doesn't look pretty

Reply:
Done!